msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/Error.kt

```diff
@@ -77,6 +77,12 @@ internal class ErrorTypes {
          */
         const val INVALID_USERNAME = "invalid_username"
 
+        /*
+        * The INVALID_SCOPES value indicates the scopes provided by the user are not valid
+        * If this occurs, valid scopes should be resubmitted
+        */
+        const val INVALID_SCOPES = "invalid_scopes"
```

Comment:
This is not a shared/common error, right? If so and if you agree with my other comment, please move this out of the common error class and into the one that's specific for this scenario.

Reply:
This is not shared now but should be as we should return similar error from signin method.

Reply:
I have moved the variable to `GetAccessTokenErrorTypes` class